es-ds-docs/pages/molecules/segmented-control.vue

```diff
@@ -0,0 +1,155 @@
+<script setup lang="ts">
+const viewActiveIndex = ref(0);
+const viewItems = [
+    {
+        label: 'List view',
+    },
+    {
+        label: 'Compare view',
+    },
+];
+
+const timeframeActiveIndex = ref(1);
+const timeframeItems = [
+    {
+        label: '90 days',
+    },
+    {
+        label: '3 months',
+    },
+    {
+        label: '1 year',
+    },
+];
+
+const esSegmentedControlProps = [
+    [
+        'ariaLabel',
```

Comment:
What does this prop correspond to? It seems like it's not being used or referenced anywhere, and the `label` prop is the one that would be the one for people to set.

Reply:
Good catch - `label` was originally called `ariaLabel` but type checking complained that passing in `:aria-label="something"` didn't match `ariaLabel` in camel case, which was really annoying, so I just simplified the name to `label` and forgot to update the props table to match.